miasm2/core/asmblock.py

```diff
@@ -91,7 +53,7 @@ class AsmConstraint(object):
 
     def __init__(self, label, c_t=c_to):
```

Comment:
`label` can be renamed to reflect its nature (`loc_key`)

Reply:
done